Swift/SwiftClosures.playground/Pages/In function.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,35 @@
+//: [Simple closure](@previous)
+
+func repeatFunc(num: Int, f: () -> ()) {
+    for _ in 1...num {
+        f()
+    }
+}
+repeatFunc(num: 5) {
+    print("Hello world")
+}
+
+
+
+var square: (Int) -> (Int) = { x in
+    return x * x
+}
+
+let myArray = [0,1,2,3,4]
```

Comment:
Styling mark: I don't like names like `myArray` and I'll advice to add spaces between array's items (like `[0, 1, 2...]`)